docs/apis/_files/db-install-xml.mdx

```diff
@@ -0,0 +1,7 @@
+The `install.xml` file is used to define any database tables, fields, indexes, and keys, which should be created for a plugin during its initial insatllation.
+
+:::note
```

Comment:
```suggestion
:::caution
```

Reply:
I think that's quite important, so it will be better to mark using some other admonition, like caution.